brave/brave-core
# Moves WalletProperties to mojo

Resolves https://github.com/brave/brave-browser/issues/4939

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- make sure that you can create wallet
- make sure that you can claim grant
- make sure that one time tip works

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
